pyomo/contrib/pyros/util.py

```diff
@@ -1654,6 +1828,47 @@ def setup_working_model(model_data, user_var_partitioning):
     delattr(original_model, temp_util_block_attr_name)
     delattr(working_model.user_model, temp_util_block_attr_name)
 
+    uncertain_param_var_idxs = []
+    for idx, obj in enumerate(working_model.orig_uncertain_params):
+        if isinstance(obj, VarData):
+            obj.fix()
+            uncertain_param_var_idxs.append(idx)
```

Comment:
Do you need to make sure that the Var has a value here?  We allow Vars with lower == upper to be uncertain Params even if they don't have a current (numeric) value, correct?

Reply:
Great! Thanks for clarifying